shared/components/layout/layout.component.js

```diff
@@ -0,0 +1,82 @@
+/*global module Promise document window*/
+
+import React from 'react';
+
+import StateManager from './../../lib/state_manager/state_manager';
+import Router from './../../lib/router/router';
+//import RouteBase from './../../lib/routes/route.base';
+import ExampleBase from './../../models/example/example.base';
```

Comment:
@senaduka When I include this import statement, I get a weird error in the browser (ie Chrome) where `RouteBase` is undefined in the Route subclasses - but the client tests still pass. I think it has something to do with webpack config.

LMK if you can figure it out.